### PR TITLE
Rubric for scoring applications for open instructor training.

### LINKED ISSUE #336: Rubric for applications for open instructor training
Please vote by leaving :+1: or :-1: on this issue.  (Click on the smiley face in the upper right above this text to leave an emoji, or add a comment.)

----

## Background

1.  There are many more applications than places, so we have to select people somehow. As an open organization, we should make our criteria explicit and open.

2.  You can view the existing application form at https://amy.software-carpentry.org/workshops/request_training/.

## Rubric

1.  Clearer explanation of what "Group Name" means (move help text up).
    1.  "If you are applying at the same time as friends or colleagues, pick a name for your group (e.g., the name of your institution) and fill that in, and we will try to put you in the same training class."
2.  Location:
    1.  Award +1 for outside Europe/UK/US/Canada/Australia/New Zealand
    2.  *Or* award +1 for being in smaller/remote/less affluent institution within EU/UK/US/Can/Aus/NZ
    3.  We will have to evaluate this manually: online university rankings and listings of school sizes won't do what we want.

3.  Domain
    1.  Award +1 for being in economics or social sciences, arts, humanities, or library science (domains where we wish to expand).

4.  Gender
    1.  Keep question as is (female, male, other, prefer not to say).
    2.  Do *not* score directly on this.

5.  Add a new question: Are you a member of one or more groups that are under-represented in computing (e.g., women, minorities, LGBT, disabled)?
    1.  Options are "No" and "Yes - specify [___]"
    2.  Score manually.
    3.  Help text will remind them that their answers are kept private under the terms of our data privacy policy (with link), and will not be shared with anyone without their explicit prior consent.

6.  New question: "Have you participated in other volunteer or non-profit groups or organizations?  (Example could include youth sports, clubs, religious or civic organizations, etc.)"

7.  Previous involvement
    1.  Change question from checkboxes to "How often have you..." and allow them to fill in numbers.
    2.  Score +1 for each previous involvement up to a maximum bonus of +3.

8.  Previous training in teaching
    1.  Change "Previous training in teaching" to "Previous training as a teacher or instructor".
    2.  +1 for "a certification or short course" or "a full degree"

9.  Previous experience in teaching
    1.  Change options to "a few hours", "a workshop", "TA for full course", or "primary instructor of full course"
    2.  Award +1 for "TA for full course" or "primary instructor for full course".

10. How frequently do you program?
    1.  Change wording to, "How frequently do you work with the tools that Data Carpentry and Software Carpentry teach, such as R, Python, MATLAB, Perl, SQL, OpenRefine, and the Unix Shell?"
    2.  Award +1 for "every day" or "a few times a week".

11. How often would you expect to teach?
    1.  Applicants must tick a box agreeing to teach within 12 months of training as part of the application process.
    2.  The purpose of this question was therefore to identify people from partner sites who are taking the training, but don't intend to teach.
    3.  We have decided *not* to have people from partner sites fill in this form, so the "Not at all" option can be removed.
    4.  Remove "Primary occupation" option as well - subsumed by "Several times a year".

12. Discretionary
    1.  Anything from -3 to +3 based on awful or awesome answers in the free-text boxes, to the question about smaller/more remote institutions, diversity, etc.

### PR DESCRIPTION
**Pull request by gvwilson**
This closes #336 and incorporates feedback given on that issue.

### CODE REVIEW

**Comment by ErinBecker** [comment]
@gvwilson This looks great to me! Thanks for pulling this together.

File: files/rubric.txt [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+
+07. Affiliation: [____________________]
+
+08. Location: [____________________]
+  Please give city, and province or state if applicable
+
+09. Country: [--pulldown--]
+
+10. [_] This a smaller, remote, or less affluent institution.
```
Comment (lexnederbragt) [inline files/rubric.txt]: Chances are people in the relevant areas may not know what the word 'affluent' means. Perhaps use 'a smaller or remote institution, or one with little funds'

File: files/rubric.txt [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+
+12. [_] I self-identify as a member of a group that is under-represented
+        in research and/or computing, e.g., women, ethnic minorities, LGBTQ,
+        etc.
+Details: [____________________]
+
+13. [_] I have been an active contributor to other volunteer or
+        non-profit groups with significant teaching or training components.
+  Data and Software Carpentry
```
Comment (lexnederbragt) [inline files/rubric.txt]: Why is this line here?
